2_Code/src/uart.h

```diff
@@ -1,22 +1,26 @@
-// This is a guard condition so that contents of this file are not included
-// more than once.
-#ifndef UART_LIB
-#define UART_LIB
+/* 
+ * Date: 03/30/2024
+ * Course number: EE 2361
+ * Term: Spring
+ * Lab/assignment number: Final Project
+ * Short Program Description: UART Library to write and read to Grove - GPS Air530.
+ */
 
-#ifndef FCY
-#define FCY 16000000L
-#endif
+#ifndef UART_H
+#define	UART_H
 
-#ifdef __cplusplus
-extern "C"
-{
-#endif /* __cplusplus */
+#ifdef	__cplusplus
+extern "C" {
+#endif
 
-    void initUART(int baudRate);
-    void sendPackage(char data[]);
+void send_GPS_Str_command(unsigned char* s);
+void send_GPS_Char_command(unsigned char command);
+unsigned char get_GPS_data(void);
+void init_UART(baudRate);
```

Comment:
Same as earlier comment, should specify what `baudRate` is (likely `unsigned int`).